docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,79 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer that installs EVE-OS on the
+underlying edge node and verifies its compatibility. It checks if the necessary
+drivers are present, tests a static and dynamic networking configurations,
+tests the available storage devices, and tests if EVE-OS can spawn a guest
+successfully. Verification is currently supported on x86 edge nodes.
+
+## Running the verification image from boot media
+
+Similar to EVE-OS installer image (see [deployment](DEPLOYMENT.md)), we:
+
+1. Produce a disk-based installer image (e.g., by running `make verification-raw`, which creates the raw image file for verification, similar to what `make installer-raw` does).
```

Comment:
I think the normal case is the docker run below, and not building it from source. (You can put building it from source in a section at the end).